app/src/main/java/org/mozilla/rocket/content/travel/data/TravelRemoteDataSource.kt

```diff
@@ -67,8 +70,24 @@ class TravelRemoteDataSource : TravelDataSource {
         TODO("not implemented")
     }
 
-    override suspend fun getCityHotels(name: String): Result<BcHotelApiEntity> {
-        TODO("not implemented")
+    override suspend fun getCityHotels(cityId: String, offset: Int): Result<BcHotelApiEntity> = withContext(Dispatchers.IO) {
+        return@withContext safeApiCall(
+                call = {
+                    if (offset % BOOKING_COM_HOTELS_OFFSET_BASE != 0) {
```

Comment:
Can replace the guard clause to kotlin's function call `require(offset % BOOKING_COM_HOTELS_OFFSET_BASE == 0) { "Offset is not multiple of 100, which means end is reached" }`